charts/wildfly/README.md

```diff
@@ -62,11 +62,28 @@ build:
   enabled: false
 ```
 
+If the application image comes from a private registry that requires authentication, you must first create a secret that contains the credentials (as explained in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)) and reference it from the `deploy.imagePullSecrets` field:
+
+```yaml
+image:
+  name: quay.io/my-private-group/my-private-image
+build:
+  enabled: false
+deploy:
+  imagePullSecrets:
+    - name: my-secret-quay-credentials
+```
+
 ## Prerequisites
 Below are prerequisites that may apply to your use case.
 
-### Pull Secret
-You will need to create a pull secret if you need to pull an image from an external registry that requires authentication. Use the following command as a reference to create your pull secret:
+### Pull Secret to Build the Application Image
```

Comment:
Maybe this section, or a dedicated section after "Push Secret to Push the Application Image", should mention the ability to pull an image for the deploy section from a private registry.

It could be enough as it is, it is just we are describing some use cases that are related to the new feature and it could be clearer for the reader that you can also pull to the deploy section an image that is in a private registry.

Reply:
That's a good suggestion. I should group all this docs in a single section "Using Private Image Registries" that explain how to pull & push the base and application images.

I'll update this PR to fix it

Reply:
@yersan I've amended the documentation to group all the doc related to secrets to manage the images in a single section

Reply:
@jmesnil Approved from my side, I read it, and looks much clear now.